.github/workflows/test.yml

```diff
@@ -0,0 +1,22 @@
+name: test
+
+on: [push]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
```

Comment:
Not sure if we can go with ubuntu, is there any options for raspbian?

Reply:
currently not supported yet.
ref. https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources